cpp/include/cudf/join.hpp

```diff
@@ -34,10 +34,11 @@
 namespace cudf {
 
 // forward declaration
-namespace detail {
+namespace hashing::detail {
 template <typename T>
 class MurmurHash3_32;
```

Comment:
Should internal uses of hash functions like this use the MurmurHash3 explicitly or should they use the “default” hash function?

Reply:
This turned out to be more involved so I will fix this in a follow-up PR.